server/models/services/CollectionJobService.js

```diff
@@ -350,6 +355,12 @@ const scheduleCollectionJob = async (
         COLLECTION_JOB_ATTRIBUTES,
         options
     );
+
+    if (process.env.AUTOMATION_CALLBACK_FQDN) {
+        await createGithubWorkflow({ job, directory, gitSha });
```

Comment:
Can we catch exceptions and cancel the collection job?

Reply:
I.E. if the create job fails set collection job to canceled immediately? If I understood correctly, this seems smart! Happy to do

Reply:
Exactly!

Reply:
Wrapped up a try/catch point for this in the `triggerWorkflow` helper